src/js/hooks/list-pages/outbound/useOutboundFilters.jsx

```diff
@@ -35,10 +35,12 @@ const useOutboundFilters = (isRequestsList) => {
   useEffect(() => {
     // TODO: When having full React, if once fetched, fetch only if a current language differs
     // TODO: from the language, that we were fetching this for
-    dispatch(fetchRequisitionStatusCodes());
     dispatch(fetchShipmentTypes());
   }, [currentLocale]);
 
+  useEffect(() => {
+    dispatch(fetchRequisitionStatusCodes(currentLocation?.id, isRequestsList));
+  }, [currentLocale, currentLocation, isRequestsList]);
```

Comment:
When we have different statuses on the dropdown, we should fetch them when:
- We are changing location (second location can not support requests approval, so this options should not be visible)
- We are changing locale (different translations)
- Changing list pages